awxkit/awxkit/api/pages/page.py

```diff
@@ -191,7 +191,7 @@ def page_identity(self, response, request_json=None, ignore_json_errors=False):
             data = response.json()
         except ValueError as e:  # If there was no json to parse
             data = dict()
-            if (response.text and not ignore_json_errors) or response.status_code not in (200, 202, 204):
```

Comment:
@wenottingham originally added this to accommodate the old endpoint (which only returned plain text).